koin-projects/koin-core/src/main/kotlin/org/koin/core/Koin.kt

```diff
@@ -228,9 +228,10 @@ class Koin {
      * Get or Create a Scope instance
      * @param scopeId
      * @param qualifier
+     * @param source
      */
-    fun getOrCreateScope(scopeId: ScopeID, qualifier: Qualifier): Scope {
-        return _scopeRegistry.getScopeOrNull(scopeId) ?: createScope(scopeId, qualifier)
+    fun getOrCreateScope(scopeId: ScopeID, qualifier: Qualifier, source: Any? = null): Scope {
```

Comment:
For what the source param is responsible for?

Reply:
source is the origin instance that can hosted by the scope. Like `scope<MyActivity>` will have an `MyActivity` instance.